assets/js/components/data/index.js

```diff
@@ -255,10 +226,48 @@ const dataAPI = {
 			// Resolve any returned data requests, then re-request the remainder after a pause.
 		} ).catch( ( err ) => {
 			// Handle the error and give up trying.
-			console.log( 'error', err ); // eslint-disable-line no-console
+			console.warn( 'Error caught during combinedGet', err ); // eslint-disable-line no-console
 		} );
 	},
 
+	handleWpError( error ) {
+		const { code, data } = error;
+
+		if ( ! code || ! data ) {
+			return;
+		}
+
+		// eslint-disable-next-line no-console
+		console.warn( 'WP Error in data response', error );
+
+		let addedNoticeCount = 0;
+
+		// Add insufficient scopes warning.
+		if ( [ 'authError', 'insufficientPermissions' ].includes( data.reason ) ) {
+			addFilter( 'googlesitekit.ErrorNotification',
+				'googlesitekit.AuthNotification',
+				fillFilterWithComponent( DashboardAuthAlert ), 1 );
+			addedNoticeCount++;
+		}
+
+		// Insufficient access permissions.
+		if ( 'forbidden' === data.reason ) {
```

Comment:
Nit-picking, but this could be `else if`.

Reply:
It could be but the comment placement becomes a bit awkward in that case. I opted to keep it this way for consistency with the first.